.github/workflows/vulnerabilites-scanner-quay.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: Fetch security scan results from quay
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "images/**"
+  workflow_dispatch:
+
+jobs:
+  build-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Fetch results
+        id: fetch-results
+        run: |
+          ./test/images/clair-scan/scan.sh
+          echo "::set-output name=VULNERABILITIES_EXIST::$VULNERABILITIES_EXIST"
```

Comment:
Because you're executing scan.sh, I don't expect `$VULNERABILITIES_EXIST` to be set once you exit the script. I would change the script to exit with a non-0 code when something went wrong and use that instead.